File: 705-Design-HashSet.py
class MyHashSet:
    def __init__(self):
        self.mySet = set()        

    def add(self, key: int) -> None:
        self.mySet.add(key)

    def remove(self, key: int) -> None:
        if key in self.mySet:
            self.mySet.remove(key)

    def contains(self, key: int) -> bool:
        return key in self.mySet

File: neetcode/705-Design-HashSet.ts
class MyHashSet {
  private set: Set<number>

  constructor() {
    this.set = new Set<number>()
  }

  add(key: number): void {
    this.set.add(key)
  }

  remove(key: number): void {
    this.set.delete(key)
  }

  contains(key: number): boolean {
    return this.set.has(key)
  }
}